os/kernel/binary_manager/binary_manager_load.c

```diff
@@ -436,7 +436,9 @@ static int loadingall_thread(int argc, char *argv[])
  ****************************************************************************/
 static int reloading_thread(int argc, char *argv[])
 {
+#ifdef CONFIG_SUPPORT_COMMON_BINARY
 	int ret;
```

Comment:
How about moving this to after line 450?

Reply:
Done